scripts/aznfs_install.sh

```diff
@@ -75,6 +115,93 @@ use_dnf_or_yum()
     fi
 }
 
+is_new_version_available()
+{
+    local current_version_integer=$(echo $1 | tr -d ".")
+    local latest_version_integer=$(echo $2 | tr -d ".")
+
+    [ $latest_version_integer -gt $current_version_integer ]
+}
+
+# Function to perform AZNFS updates
+perform_aznfs_updates() 
+{
+    # For watchdog, the flag file will always be present; otherwise, the service will be "manual-update"
+    if [ -f /tmp/update_in_progress_from_watchdog.flag ] || [ "$RUN_MODE" == "manual-update" ]; then
+        if [ "$install_cmd" == "apt" ]; then
+            AZNFS_RELEASE="aznfs-${RELEASE_NUMBER}-1"
+            package_name=${AZNFS_RELEASE}_amd64.deb
+        elif [ "$install_cmd" == "zypper" ]; then
+            AZNFS_RELEASE_SUSE="aznfs_sles-${RELEASE_NUMBER}-1"
+            package_name=${AZNFS_RELEASE_SUSE}.x86_64.rpm
+        else
+            AZNFS_RELEASE="aznfs-${RELEASE_NUMBER}-1"
+            package_name=${AZNFS_RELEASE}.x86_64.rpm
+        fi
+        
+        wget "https://github.com/Azure/AZNFS-mount/releases/download/${RELEASE_NUMBER}/${package_name}" -P /tmp
```

Comment:
wget can likely fail, so we need to check the return status and also check for presence of a valid "/tmp/${package_name}" before going ahead with installation.
If wget fails, a useful error log would help.